docs/release-notes/august-2023.md

```diff
@@ -0,0 +1,157 @@
+- **Julia:** Type information from declarations can now be used in metavariable-type. For instance, the program:
+  ```
+  x :: Int64 = 2
+  ```
+  now allows uses of x to match to the type Int64. ([#8470](https://github.com/returntocorp/semgrep/issues/8470))
+- **Julia:** Metavariables are now able to appear anywhere that identifiers can. For instance, they were not able to appear as the argument to a `do` block. ([#8486](https://github.com/returntocorp/semgrep/issues/8486)) Now, you can write patterns such as:
+```
+map($Y) do $X
+  ...
+end
+```
+
+- **Java:** Fixed naming bug affecting Java and other OO languages that allowed a method parameter to shadow a class attribute. For example, in:
+  ```
+  class Test {
+  
+      private int x;
+  
+      public void test2(int x) {
+          foo(this.x);
+      }
+  
+  }
+  ```
+  Semgrep was considering that `this.x` referred to the parameter `x` of `test2` rather than to the class attribute `x`. ([#8508](https://github.com/returntocorp/semgrep/pull/8508))
+
+
+### Changed
+
+- Running the `semgrep`  command now displays the help message. Previously, the `semgrep` command ran a SAST scan.
+
+### Removed
+
+- `python -m semgrep` has been removed. Instead, invoke Semgrep directly by entering `semgrep` in the CLI.
+- Semgrep no longer looks for a `.semgrep.yml` config file or `.semgrep/` in the current directory, which previously caused conflicts when invoking `semgrep` from your home directory. This is because the home directory can contain a `.semgrep/settings.yml` file that is not a Semgrep rule. ([#4457](https://github.com/returntocorp/semgrep/issues/4457))
+  - The preferred method to run rules is to explicitly pass rules through the `--config` option. For example, to run a `.semgrep.yml` file containing rules, you must enter `semgrep --config .semgrep.yml`.
+- If you previously wrapped Semgrep Python code by calling `semgrep_main.main`, you must replace the previous call with `run_scan.run_scan`. Note that these Python calls will be removed in the future.
+- `--enable-metrics` and `--disable-metrics` have been removed. Instead, use any of the following:
+  - `--metrics=on`
+  - `--metrics=off`
+  - `--metrics=auto`
+
+## Semgrep Pro Engine
+
+<!-- Separated pro engine because it is proprietary -->
+
+### Fixed
+
+- **JavaScipt (JS) or TypeScript (TS) taint mode:** fixed a bug introduced in 1.33.1 that had the side-effect of hurting performance of taint rules on JS or TS repositories that used destructuring in a function's formal parameters.
+
+## Semgrep Cloud Platform
+
+### Added
+
+- The `semgrep ci` command now displays enabled products when scans are command now created or when the scan config is generated from Semgrep Cloud Platform. Additionally, if no products are enabled then a friendly error is raised and the scan is stopped. You must enable a product to start a scan.
```

Comment:
```suggestion
- The `semgrep ci` command now displays enabled products when the scan config is generated from Semgrep Cloud Platform. Additionally, if no products are enabled then a friendly error is raised and the scan is stopped. You must enable a product to start a scan.
```

It seems weird to me to have this in the SCP section since it's a change in Semgrep behavior, even if it only affects `semgrep ci`. Thoughts?

Reply:
Just curious - what seems weird about it to you? My thought was: `semgrep ci` can run any, all, or none of the products, but does require an SCP account and isn't under Semgrep OSS.

Reply:
That seems reasonable. It stuck me as odd, but logically I think your reasoning is better than my instinct 😄